packages/shader-shaderlab/src/shaders/PBR.gs

```diff
@@ -58,6 +58,10 @@ Shader "PBR.gs" {
       Header("Transmission"){
         material_Transmission("Transmission", Range(0, 1, 0.01)) = 0;
         material_TransmissionTexture("TransmissionTexture", Texture2D);
+        material_AttenuationColor("AttenuationColor", Color ) = (1, 1, 1, 1);
```

Comment:
Transmission is 0.01 in editor, maybe default 0 is better（Only a few materials need to be enabled）